app/notifications/rest.py

```diff
@@ -160,6 +161,16 @@ def get_notification_return_data(notification_id, notification, template):
     return output
 
 
+def _service_can_send_internationally(service, number):
```

Comment:
This response code matches that used with team key errors. Interesting to make it a 403 - ie forbidden to do the action, not the request was bad.

Think this is better (403) will put a story in for it.